src/backend/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1684,26 +1684,31 @@ msgstr "<span class=\"student-name\">%(student)s</span><br/> a acquis les compé
 msgid "Issued on %(creation_date)s in France."
 msgstr "Délivré le %(creation_date)s en France."
 
-#: joanie/core/templates/issuers/unicamp-degree.html:50
+#: joanie/core/templates/issuers/unicamp-degree.html:52
 msgid "Educational coordinator "
 msgstr "Responsable pédagogique "
 
-#: joanie/core/templates/issuers/unicamp-degree.html:70
+#: joanie/core/templates/issuers/unicamp-degree.html:60
+msgid "Issued by"
+msgstr ""
+
+#: joanie/core/templates/issuers/unicamp-degree.html:76
 msgid "Certification level"
 msgstr "Niveau de certification"
 
-#: joanie/core/templates/issuers/unicamp-degree.html:77
+#: joanie/core/templates/issuers/unicamp-degree.html:84
 #, python-format
 msgid "\n"
-"                        This micro-certification is issued in compliance with the common framework for micro-certifications:\n"
-"                        <br /><a href=\"%(microcertification_terms_url)s\">%(microcertification_terms_url)s</a>\n"
-"                        "
-msgstr "\n"
-"                        Cette micro-certiﬁcation est délivrée en conformité avec le cadre commun des micro-certiﬁcations :\n"
-"                        <br /><a href=\"%(microcertification_terms_url)s\">%(microcertification_terms_url)s</a>\n"
-"                        "
+"                            This micro-certification is issued in compliance with the common framework for micro-certifications:\n"
+"                            <br /><a href=\"%(microcertification_terms_url)s\">%(microcertification_terms_url)s</a>\n"
+"                            "
+msgstr ""
```

Comment:
FMI, why is this translation removed ?

Reply:
Because this is a draft text which needs to be cleaned. This like the other bolo text...